Dockerfile

```diff
@@ -20,6 +32,18 @@ COPY --chown=ubuntu:ubuntu ./Makefile ./Makefile
 RUN mkdir -p .eval-dev-quality
 RUN mkdir -p /app/evaluation
 
+# Install Ruby
+RUN mkdir -p /tmp/compile
+RUN wget https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.4.tar.gz && \
+	tar -xf ruby-3.3.4.tar.gz -C /tmp/compile/ && \
+	rm ruby-3.3.4.tar.gz
+WORKDIR /tmp/compile/ruby-3.3.4
+RUN ./configure --prefix /app/.eval-dev-quality/ruby-3.3.4 --disable-install-doc
+RUN	make install
```

Comment:
And that now takes 8 Minutes, right? :see_no_evil: no prebuilt ruby... like ubuntu package? Guess that one is outdated.

Reply:
Added the points into their own Issue so we do keep track: https://github.com/symflower/eval-dev-quality/issues/319

Reply:
Yeah artifacts would be stuff you want to pass from one Job to the next in the CI